src/features/chat/components/ChatTopBar.tsx

```diff
@@ -0,0 +1,51 @@
+import { UserProfileWidget } from "shared/components";
+import useTheme from "features/theme/useTheme";
+import { Box, IconButton } from "@mui/material";
+import { Phone, Videocam, Info } from "@mui/icons-material";
+
+type ChatTopBarProps = {
+  profilePicture?: string;
```

Comment:
I would update this to profilePic so we don't have to transform the property name we get from the response.
Also I think this shouldn't be an optional prop, since we're going to have default avatar for users.